packages/shared/src/components/filters/FeedFilters.tsx

```diff
@@ -35,6 +35,7 @@ const unBlockPromptOptions: PromptOptions = {
   },
 };
 
+// TODO-AS-814 remove and replace instances with FeedSettingsEdit
```

Comment:
@capJavert This is still open?

Reply:
e4e424ff316649869967a0f06bbbc25a599a2c67